src/api/index.ts

```diff
@@ -42,7 +43,7 @@ const posts = {
 }
 
 const post = {
```

Comment:
export here instead of export at the bottom. And export api should be default. Same thing with other request